Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln

```diff
@@ -1,10 +1,10 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
```

Comment:
Visual Studio Version should be 17. You need to resave the solution in the new version.